### PR TITLE
Add list of metrics to check in Cloud integrations

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Add a list (as a partial) of metrics and tags to check with Cloud integrations. 

# Links

Fixes https://github.com/timescale/docs-private/issues/50

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Comment by Loquacity** [comment]
Datadog:

image

CloudWatch:

image

**Comment by lukas-bernert** [comment]
(1) Is this how we want to display the tags? e.g. should `host = <timescale-region>.timescale.cloud` be one variable?
`host` is the tag, `<timescale-region>-timescale.cloud` is just an example of what the variable looks like.

Similarly, if we use the `=` for host we should do the same with e.g. `region = <timescale-region>`.

Sorry, I might have been unclear with my examples in the original issue.
I figured our docs have certain standards on how to display variables and possible values/examples.

If what you're proposing is fine with y'all, ignore my comment :) 


(2) Do we want to display the same information twice? or is it enough if we display it in a place where it applies for both DD and CW?

**Comment by lukas-bernert** [comment]
re: what I mean by standards.

See the API reference for example.
We always use a table for parameters. In t
Screenshot 2022-09-12 at 09 44 15
his case, "tags" would be similar to API params.

File: _partials/_cloud-integrations-metrics.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+names are:
+
+- `timescale.cloud.system.cpu.usage.millicores`
+- `timescale.cloud.system.cpu.total.millicores`
+- `timescale.cloud.system.memory.usage.bytes`
+- `timescale.cloud.system.memory.total.bytes`
+- `timescale.cloud.system.disk.usage.bytes`
+- `timescale.cloud.system.disk.total.bytes`
+
+Additionally, Timescale Cloud exports tags that you can use to filter your results. You can also check that these tags are being correctly exported:
```
Comment (Rajakavitha1) [inline _partials/_cloud-integrations-metrics.mdx:12]:
```suggestion
Additionally, Timescale Cloud exports tags that you can use to filter your results.
You can also check that these tags are being correctly exported:
```

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!!

**Comment by Loquacity** [comment]
> (2) Do we want to display the same information twice? or is it enough if we display it in a place where it applies for both DD and CW?

This is why I decided to do it as a snippet: we only write it once, but it appears in the most logical spot for each integration.

**Comment by Loquacity** [comment]
**Datadog:**
image

**Cloudwatch:**
image

**Review by lukas-bernert — APPROVED** [review]

**Pull request merged**